.travis.yml

```diff
@@ -22,19 +22,20 @@ before_script:
     fi
 script:
   - |
-    travis-cargo test -- --verbose --no-default-features &&
+    travis-cargo --only stable test -- --verbose --no-default-features &&
     travis-cargo --skip nightly test -- --verbose --features "yaml unstable" &&
-    travis-cargo --only nightly test -- --verbose --features "yaml unstable nightly" &&
-    travis-cargo --only nightly bench
```

Comment:
In case you're not aware, you can do `cargo bench --no-run` to compile benchmarks without running them. Might be a useful middle ground!

Reply:
Ah ok, I didn't know that, thanks! That is exactly what I'd like to do, for some reason I thought that was only possible with tests.